src/components/site/tableOfContents/table-of-contents.module.scss

```diff
@@ -0,0 +1,36 @@
+:global {
+  @include react-md-toolbars;
```

Comment:
probs not necessary if not sticking a toolbar inside the drawer?

Reply:
ah, yes. just need a little clean up